app/cdap/components/WranglerGrid/TransformationToolbar/iconStore.tsx

```diff
@@ -0,0 +1,225 @@
+    <path
+      d="M13 12.5C13 11.9477 12.5523 11.5 12 11.5C11.4477 11.5 11 11.9477 11 12.5C11 13.0523 11.4477 13.5 12 13.5C12.5523 13.5 13 13.0523 13 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M6 12.5C6 11.9477 5.55228 11.5 5 11.5C4.44772 11.5 4 11.9477 4 12.5C4 13.0523 4.44772 13.5 5 13.5C5.55228 13.5 6 13.0523 6 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M20 12.5C20 11.9477 19.5523 11.5 19 11.5C18.4477 11.5 18 11.9477 18 12.5C18 13.0523 18.4477 13.5 19 13.5C19.5523 13.5 20 13.0523 20 12.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const GridIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12 3.5H19C19.5304 3.5 20.0391 3.71071 20.4142 4.08579C20.7893 4.46086 21 4.96957 21 5.5V19.5C21 20.0304 20.7893 20.5391 20.4142 20.9142C20.0391 21.2893 19.5304 21.5 19 21.5H12M12 3.5H5C4.46957 3.5 3.96086 3.71071 3.58579 4.08579C3.21071 4.46086 3 4.96957 3 5.5V19.5C3 20.0304 3.21071 20.5391 3.58579 20.9142C3.96086 21.2893 4.46957 21.5 5 21.5H12M12 3.5V21.5"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M3 12.492H20.7689"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const SearchIconn = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clipPath="url(#clip0_1939_22386)">
+      <path
+        d="M15.755 14.7559H14.965L14.685 14.4859C15.665 13.3459 16.255 11.8659 16.255 10.2559C16.255 6.66586 13.345 3.75586 9.755 3.75586C6.165 3.75586 3.255 6.66586 3.255 10.2559C3.255 13.8459 6.165 16.7559 9.755 16.7559C11.365 16.7559 12.845 16.1659 13.985 15.1859L14.255 15.4659V16.2559L19.255 21.2459L20.745 19.7559L15.755 14.7559ZM9.755 14.7559C7.265 14.7559 5.255 12.7459 5.255 10.2559C5.255 7.76586 7.265 5.75586 9.755 5.75586C12.245 5.75586 14.255 7.76586 14.255 10.2559C14.255 12.7459 12.245 14.7559 9.755 14.7559Z"
+        fill="black"
+        fillOpacity="0.54"
+      />
+    </g>
+    <defs>
+      <clipPath id="clip0_1939_22386">
+        <rect width="24" height="24" fill="white" transform="translate(0 0.5)" />
+      </clipPath>
+    </defs>
+  </svg>
+);
+
+export const Expand = (
+  <svg width="12" height="9" viewBox="0 0 12 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M10.59 8.20508L6 3.62508L1.41 8.20508L0 6.79508L6 0.795078L12 6.79508L10.59 8.20508Z"
+      fill="black"
+      fillOpacity="0.54"
+    />
+  </svg>
+);
+
+export const menuArrowIcon = (
```

Comment:
Use Mui `ChevronRight`?

Reply:
Replaced it with ChevronRightRoundedIcon .